<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEmbedForm</class>
 <widget class="QWidget" name="ImageEmbedForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Внедрение сообщения в картинку</string>
  </property>
  <widget class="QLabel" name="label_message">
   <property name="text">
    <string>Сообщение:</string>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageField">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="text">
    <string>Картинка:</string>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="imagePathField">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="text">
    <string>Обзор</string>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="embedButton">
   <property name="text">
    <string>Внедрить</string>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
